### DISCUSSION TITLE
game freezes while searching in song select, can lead to crashes

### DISCUSSION THREAD

**Discussion by Tinchozz** [ppy/osu#15613] [Q&A] [answered]
Video describing the issue:
https://user-images.githubusercontent.com/93566809/141661036-23c383b8-995f-4b44-900e-62e5e6f563ce.mp4
[database.log](https://github.com/ppy/osu/files/7532874/database.log)
[network.log](https://github.com/ppy/osu/files/7532875/network.log)
[performance.log](https://github.com/ppy/osu/files/7532876/performance.log)
[performance-audio.log](https://github.com/ppy/osu/files/7532877/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/7532878/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/7532879/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/7532880/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/7532881/runtime.log)
[updater.log](https://github.com/ppy/osu/files/7532882/updater.log)

**Comment by integerrr** [comment, marked as answer]
#11571
